lib/src/interfaces/INotification.dart

```diff
@@ -19,61 +19,39 @@ part of puremvc;
  *
  * See [IView], [IObserver]
  */
-abstract class INotification
-{
-
-    /**
-     * Get the [name] of the [INotification].
-     *
-     * -  Returns [String] - the name of the [INotification].
-     */
-    String getName();
-
-    /**
-     * Set the [body] of the [INotification].
-     *
-     * -  Param [body] - the body of the [INotification].
-     */
-    void setBody( Object body );
-
-    /**
-     * Get the [body] of the [INotification].
-     *
-     * -  Returns [Dynamic] - the body of the [INotification].
-     */
-    dynamic getBody();
-
-    /**
-     * Set the [type] of the [INotification].
-     *
-     * -  Param [type] - the type of the [INotification].
-     */
-    void setType( String type );
-
-    /**
-     * Get the [type] of the [INotification].
-     *
-     * -  Returns [String] - the type of the [INotification].
-     */
-    String getType();
-
-    /**
-     * This [INotifications]'s [body]
-     */
-    void set body( dynamic bodyObject );
```

Comment:
Because we have methods: `getBody`, `setBody`, the accessors looks excess, they duplicate functionality. And in my practice I always use these methods. What are the reasons for accessors existence?

Reply:
Accessors are not duplicate functionality, they are alternate functionality. You can say `myModel.body` or `myModel.getBody()` whichever may be your preference. They exist as part of the original framework and all ports. 

It's important to remember that personal stylistic choices should not distort a port. A port should hew as closely to the reference implementation as is practicable. This is why the ActionScript Developers Guide to PureMVC is generally useful do devs using most ports. Because the implementation and idioms are very similar across ports. If developers used to one port of the framework find that the same idioms they are used to are present in another port, they get to joy quicker. If they arrive here and find that things are different owing to stylistic choices, well, it all becomes a bit loosey-goosey. The promise of easy portability breaks down. If this one thing is arbitrarily different, what others might be?